public/index.html

```diff
@@ -90,6 +90,9 @@
     <a href="/auth/google" class="btn btn-danger">
       <span class="google-plus"></span>Google Sign in
     </a>
+  <script type="text/javascript">
+  document.getElementById("navbar").style.visibility="hidden";
```

Comment:
Hi,
This would remove navigation bar not only from login page, but also from all the views/pages.